packages/tools/viewer-alpha/public/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Viewer Local Development</title>
+
+        <style>
+            html,
+            body {
+                width: 100%;
+                height: 100%;
+                padding: 0;
+                margin: 0;
+                overflow: hidden;
+            }
+        </style>
+    </head>
+
+    <body>
+        <babylon-viewer src="https://playground.babylonjs.com/scenes/BoomBox.glb" env="https://assets.babylonjs.com/environments/ulmerMuenster.env"> </babylon-viewer>

```

Comment:
one of my learnins from the original viewer was that it is important to namespace parameters that are viewer-specific, i.e. instead of env="...", it should be something like `bv:env="..."` (or `bv-env:"..."`). Mainly because of collisions with pure html (`width`, `height`, etc'). It is also easier to process the element, because you only need to deal with the parameters marked with the prefix, the rest should be dealt with by the browser.

Reply:
Talked offline and decided to leave it as is for now (no namespacing of attributes), in part because with custom elements you have to explicitly declare the attributes you will get change notifications for.